bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -364,9 +367,11 @@ spec:
             {{- if .Values.pgpool.extraVolumeMounts }}
             {{- include "common.tplvalues.render" (dict "value" .Values.pgpool.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
+          {{- end }}
         {{- if .Values.pgpool.sidecars }}
         {{- include "common.tplvalues.render" (dict "value" .Values.pgpool.sidecars "context" $) | nindent 8 }}
         {{- end }}
+      {{- if or .Values.pgpool.configuration .Values.pgpool.configurationCM .Values.pgpool.initdbScripts .Values.pgpool.initdbScriptsCM .Values.pgpool.initdbScriptsSecret .Values.postgresql.usePasswordFile .Values.pgpool.tls.enabled .Values.pgpool.extraVolumes }}
```

Comment:
This conditional is missing the value `.Values.pgpool.usePasswordFile`.

Reply:
Updated and I also included some multi-line formatting that makes this more apparent..